recipes/cli-duckdb/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2018-2023 Stichting DuckDB Foundation
```

Comment:
No need to include this file as it is part of the source tarball.

Reply:
removed